diff --git a/dist/index.js b/dist/index.js
index ede756db9b170927f76d9023c9b0b1506ebaf04b..82b8bb45dc2e0e597197bdd5cf6281e36f521c43 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -293,7 +293,6 @@ async function convert(input, outputPaths, options = {}) {
         // will get called which bundles media files etc.
         if (outputPath === exports.STDIO_PATH) {
             const content = await dump(node, to !== null && to !== void 0 ? to : 'txt', encodeOptions);
-            console.log(content);
             if (outputPaths.length > 0)
                 return content;
             continue;
